packages/esm-service-queues-app/src/views/queue-table-by-status-view.component.tsx

```diff
@@ -0,0 +1,22 @@
+import { InlineNotification } from '@carbon/react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import { useQueue } from '../hooks/useQueue';
+import { QueueTableByStatus } from '../queue-table/queue-table-by-status.component';
+import { QueueTableByStatusSkeleton } from '../queue-table/queue-table-by-status-skeleton.component';
```

Comment:
The established pattern is to use default exports, so I'd expect to see:

```suggestion
import QueueTableByStatus from '../queue-table/queue-table-by-status.component';
import QueueTableByStatusSkeleton from '../queue-table/queue-table-by-status-skeleton.component';
```

instead.